### PR TITLE
chore: Add passing Markdownlint config

### PR DESCRIPTION
**Pull request by nschonni**
Suppressed all failing rules so editing in VS Code has less noise

### CODE REVIEW

**Comment by e0i** [comment]
@lizap @danihalfin @Dansimp Would you be able to review this PR that introduces a VS Code configuration? Thanks.

File: .markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "MD029": false,
+    "MD030": false,
+    "MD031": false,
+    "MD032": false,
+    "MD033": false,
+    "MD034": false,
+    "MD036": false,
+    "MD037": false,
+    "MD038": false,
+    "MD039": false,
```
Comment (nschonni) [inline .markdownlint.json]: Should be removed since #4045 landed

File: .markdownlint.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "MD040": false,
+    "MD041": false,
+    "MD042": false,
+    "MD045": false,
```
Comment (nschonni) [inline .markdownlint.json]: Should be removed since #4046 landed

**Comment by e0i** [comment]
@lizap @DaniHalfin @Dansimp Any updates with this PR? Thanks.

**Comment by DaniHalfin** [comment]
I'd rather keep any unnecessary config files out of the repo.

**Comment by nschonni** [comment]
@DaniHalfin if you're editing articles with VS Code and the MS Docs Authoring Pack, many articles have warnings throughout without adding this

**Comment by DaniHalfin** [comment]
Agreed, but this is a tooling config and pertains to the preferences of an individual. I don't want to push (or maintain, for that matter) that kind of configuration.

**Comment by nschonni** [comment]
The current PR isn't a preference, it just silences the currently failing rules so they can be cleaned up later. Some teams do actually go the preference route though like https://github.com/aspnet/AspNetCore.Docs/blob/master/.markdownlint.json

**Comment by DaniHalfin** [comment]
The preference part I was talking about, is whether or not to use markdown lint (even with the authoring pack installed, markdwon lint can be disabled). 
Adding and maintaining tooling configuration in a content repo isn't something we're looking to do in our content repo at this point, but we can revisit if we get more feedback to that point.

**Comment by e0i** [comment]
@nschonni 

From our understanding the pull request has been declined via **DaniHalfin**'s comments and thus it is being closed.

Thank you so far with the rest of your contributions. Much appreciated.

**Comment by e0i** [comment]
@officedocsbot close

**Pull request closed without merging**